### ISSUE TITLE
Estudiante aparece en otro curso y no se muestra progreso en curso actual

### ISSUE THREAD

**Issue by estefaniamiguel** [mumuki/mumuki-classroom#176]
Este error fue algo que vi luego de un reclamo de les estudiantes de les que no veía progreso.
Les estudiantes me indican que sí estaban haciendo ejercicios, pero yo desde el classroom no los veía. Muchos, con solo agregarlos comencé a verlos y otros, los agregué pero no veo su progreso.
Entonces pedí que me enviaran un mensaje por medio de "levantá la mano" como para saber qué pasaba.

Me viene este mensaje del estudiante Jordi
![captura de pantalla 2018-05-30 a la s 19 35 20](https://user-images.githubusercontent.com/3276427/40751403-618db5c6-6441-11e8-806c-debedf21ee54.png)

Si se fijan, se ve que Jordi estaría inscripto al cuatrimestre anterior y no al actual.

Pero, si voy al listado de estudiantes, no está:
![screencapture-algo2-undav-classroom-mumuki-io-2018-05-30-19_35_39](https://user-images.githubusercontent.com/3276427/40751440-88ba7dd2-6441-11e8-86f3-f86a053b967b.png)

El si está en el curso de este cuatrimestre:
![screencapture-algo2-undav-classroom-mumuki-io-2018-05-30-19_35_54](https://user-images.githubusercontent.com/3276427/40751452-9c68965c-6441-11e8-9122-26f7668c5180.png)
Pero sin ningún progreso (cosa que si tiene según vi en su computadora)

Qué estará pasando?
Gracias!

**Comment by aguspina** [comment]
Hola @estefaniamiguel 

El progreso que el tiene, es debido a las soluciones que envió durante el transcurso del cuatrimestre anterior. 

Cuando lo diste de alta en el nuevo curso, comienza con el progreso en 0, porque lo que el tenia resuelto esta asociado al curso anterior.

Por lo tanto, al hacerte una pregunta desde el ejercicio, classroom busca soluciones para ese ejercicio, y encuentra que las tiene en el curso viejo y NO en el nuevo, por lo tanto te redirige a las soluciones del curso viejo.

Una aclaración, el si está en el curso viejo, sólo que está desvinculado, por eso no aparece en el listado principal, pero si aparece si apretás en "mostrar".

**Comment by estefaniamiguel** [comment]
Claro, la idea es que se anotó mal, entonces al desvincular, mi idea era que pasara todo lo que hizo al nuevo mumuki o lo que hiciera de ahora en más estuviera en el nuevo mumuki.
